app/src/main/res/values/themes.xml

```diff
@@ -130,24 +152,24 @@
         <item name="android:elevation">3dp</item>
         <item name="android:background">#00FFFFFF</item>
     </style>
-    <style name="cardviewTrivia">
+    <style name="CardViewTrivia">
         <item name="android:layout_marginVertical">@dimen/spaceNormal</item>
```

Comment:
use android:layout_marginTop and marginBottom instead of android:layout_marginVertical